distribution/docker/build.gradle

```diff
@@ -105,10 +106,19 @@ task copyKeystore(type: Sync) {
   }
 }
 
-preProcessFixture {
-  if (TestFixturesPlugin.dockerComposeSupported()) {
-    dependsOn assemble
+elasticsearch_distributions {
+  Flavor.values().each { distroFlavor ->
+    "docker_$distroFlavor" {
+      flavor = distroFlavor
+      type = 'docker'
+      version = VersionProperties.getElasticsearch()
+      required = false // This ensures we skip this testing if Docker is unavailable
```

Comment:
Does this **only** skip testing? Docker is required for a successful overall build on supported OSs, no?

Reply:
This skips building the upstream docker images that are _consumed_ by the tests. I realize now that comment isn't great, I'll improve it.

Reply:
I updated the comment text to make it clear this simply makes building the images optional. The logic for actually skipping the tests themselves lives in `TestFixturesPlugin`.